src/main/java/ui/TestController.java

```diff
@@ -25,6 +25,11 @@
  * instance which can be called from tests that need to access the UI class directly.
  */
 public final class TestController {
+
+    public static final String TEST_DIRECTORY = "settings";
+    public static final String TEST_SESSION_CONFIG_FILENAME = "test.json";
+    public static final String TEST_USER_CONFIG_FILENAME = "test_user.json";
```

Comment:
I'm not sure if we should put it here. I think putting it inside `Preferences` would be better.